example/get_old_transactions.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:async';
+
+import 'package:packagekit/packagekit.dart';
+
+void main(List<String> args) async {
+  if (args.isEmpty) {
+    print('Getting all known transactions');
+  }
+  var number = int.tryParse(args.first) ?? 0;
```

Comment:
Using first will throw an exception if args is empty - instead probably want something like:
```
var number = 0;
if (args.length > 0)
{
   number = int.tryParse(args.first) ?? 0;
}
```